plonky2/src/fri/challenges.rs

```diff
@@ -28,6 +32,8 @@ impl<F: RichField, H: Hasher<F>> Challenger<F, H> {
         pow_witness: F,
         degree_bits: usize,
         config: &FriConfig,
+        final_poly_coeff_len: Option<usize>,
+        query_round_step_count: Option<usize>,
```

Comment:
I find this name unclear: maybe say that it's the max number of steps? So something like `max_number_steps`?

Reply:
renamed to `max_num_query_steps`